modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPBlueprintResourceImpl.java

```diff
@@ -296,6 +285,24 @@ private String _getSchemaVersion() {
 			contextUriInfo.getPath(), "v", StringPool.SLASH);
 	}
 
+	private void _setActions(long sxpBlueprintId, SXPBlueprint sxpBlueprint) {
```

Comment:
@gustavolimav no need to pass the ID as it's already within the SXPBlueprint. Also if you change this method to return the SXPBlueprint, you can simplify the related _setAction() calls.